templates/includes/record-revealed.html

```diff
@@ -1,21 +1,34 @@
-{% load static wagtailimages_tags wagtailcore_tags %}
-<div class="explore-intro">
-    <div class="container">
-        <div class="row">
-            <div class="col-lg-8 col-sm-12">
-                <p class="explore-intro__title explore-intro__title--meta">Record revealed</p>
-                <h1 class="explore-intro__title">{{ page.title }}</h1>
-                <div class="explore-intro__paragraph{% if display_content_warning %} explore-intro__paragraph--short{% endif %}">{{ page.intro|richtext }}</div>
-                {% if display_content_warning %}    
-                    {% include "includes/content-warning.html" with custom_warning_text=custom_warning_text %} 
-                {% endif %}
-            </div>
-            <div class="col-4 explore-intro__image-container">
-                {% if page.intro_image %}
-                    {% image page.intro_image fill-390x390 as intro_image %}
-                    <img height=390 width=390 src="{{ intro_image.url }}" alt="" class="explore-intro__image" />
-                {% endif %}
-            </div>
-        </div>
+{% load i18n wagtailimages_tags wagtailcore_tags records_tags %}
+{# TODO this is a near duplicate of template/blocks/featured_record_article.html #}
+
+<div class="featured-record-article featured-record-article--standalone"
+    data-container-name="featured-record-article">
+    {% if record.teaser_image %}
+        <a href="{{ record.record.url }}">
+            <span class="sr-only">{{ record.title }}</span>
+            <picture>
+                {% image record.teaser_image fill-626x486-c100 as mobile_img %}
+                <source media="(max-width: 1024px)" srcset="{{ mobile_img.url }}"/>
+                {% image record.teaser_image fill-390x303-c100 as desktop_img %}
+                <source media="(min-width: 1025px)" srcset="{{ desktop_img.url }}"/>
+                <img class="featured-record-article__image"
+                    height="{{ mobile_img.height }}"
+                    width="{{ mobile_img.width }}"
+                    src="{{ mobile_img.url }}"
+                    alt=""/>
+            </picture>
+        </a>
+    {% endif %}
+    <div class="featured-record-article__container">
+        <p class="featured-record-article__meta u-margin-xxs">{% trans "Record Revealed" %}</p>
+        <a href="{{ record.record.url }}" class="featured-record-article__link">
```

Comment:
Note: Looking at tabbing for this element this link does not have an `active` psudeo state. Could we add one as it's not clear for our screen reader users.

Reply:
Also, it looks like the typography styles is overidding our link colour here; 
![Screenshot from 2023-05-25 12-02-48](https://github.com/nationalarchives/ds-wagtail/assets/22497837/8d347f9f-e44f-403f-914f-16ea12bff94d)

We might need a more specific selector here to override it, or removing that blanket style which would be my preference as we mostly just override it anyway.